cms/tests/test_apphooks.py

```diff
@@ -32,7 +32,7 @@
 APP_MODULE = "cms.test_utils.project.sampleapp.cms_apps"
 
 
-class ApphooksTestCase(CMSTestCase):
+class ApphooksTestCase(ClearURLs, CMSTestCase):
```

Comment:
I've added ``ClearURLs`` to those tests cases where I noticed it. It uses ``setUpClass`` as opposed to ``setUp`` as the urls are shared for the whole test case. Further investigating shows that this might not even be enough (see #4289, maybe some ``del sys.modules[???]`` will be needed too), but it's a start. I also like having it as a mixin so you don't have random cleanup code all over the place, hopefully makes it easier to not forget it and also means we have a single place to "solve" this issue.